<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 230, 230);
</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>201</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border: none;
    background-image: url(C:/Users/compu/Documents/Qt Projects/Chat/Chat/talk.png);
     background-repeat: no-repeat;
    background-position: center;
    background-size: cover;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Iniciar Sesion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>300</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear Cuenta</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 28pt &quot;Cascadia Mono&quot;;
color: rgb(236, 118, 144);</string>
      </property>
      <property name="text">
       <string>ChatApp</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>0</y>
        <width>821</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border: none;
    background-image: url(C:/Users/compu/Documents/Qt Projects/Chat/Chat/fondo.jpg);
     background-repeat: no-repeat;
    background-position: center;
    background-size: cover;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 180, 181);</string>
      </property>
      <property name="text">
       <string>ingresar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 127);
font: 600 24pt &quot;Segoe UI&quot;;
background-color: rgb(251, 219, 222);</string>
      </property>
      <property name="text">
       <string>Iniciar Sesion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 11pt &quot;Segoe UI&quot;;
background-color: rgb(251, 219, 222);</string>
      </property>
      <property name="text">
       <string>Usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 11pt &quot;Segoe UI&quot;;
background-color: rgb(251, 219, 222);</string>
      </property>
      <property name="text">
       <string>Contrase単a</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="campo1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="campo2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(251, 219, 222);
background-color: rgb(251, 219, 222);
color: rgb(23, 23, 35);
</string>
      </property>
      <property name="text">
       <string>Olvide mi contrase単a :(</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 127);
font: 600 24pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Crear Cuenta</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre Completo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Correo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Contrase単a</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmacion de Contrase単a</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>330</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Edad</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pregunta de Seguridad</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>270</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Respuesta</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>200</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nombre de Mascota</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cancion Favorita</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="campo3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="campo4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="campo5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="campo6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="campo7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="campo8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="campo9">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>270</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color:rgb(255, 255, 255);	
	background-color: rgb(255, 78, 81);
	
	selection-background-color: rgb(95, 95, 95);
}</string>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar Avatar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>70</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="mailsCB">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>@gmail.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>@hotmail.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>@yahoo.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>@unitec.edu</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>-20</y>
          <width>621</width>
          <height>501</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>370</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>salir</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Contactos</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Chats</string>
        </property>
       </widget>
       <widget class="QLabel" name="perfil">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="userLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 600 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
